src/main/java/Deadline.java

```diff
@@ -1,7 +1,7 @@
-public class Deadlines extends Task {
+public class Deadline extends Task {
 
     protected String by;
-    public Deadlines (String description, String by) {
+    public Deadline(String description, String by) {
```

Comment:
4. Perhaps a space should be used between the names and the inputs. E.g 
Deadline (String description, String by) {
    ...
}